src/rp2_common/pico_audio_i2s/audio_i2s.pio

```diff
@@ -42,6 +42,24 @@ bitloop0:
 public entry_point:
     set x, 14         side 0b11
 
+.program audio_i2s_swapped
```

Comment:
I wonder if it might also be worth duplicating (and updating) the `; Two side-set pins are used. Bit 0 is clock, bit 1 is word select.` comment here?